### ISSUE TITLE
iron-collapse accessibility report

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-collapse#5]
The accessibility developer tools audit results are: 

"Warning] Text elements should have a reasonable contrast ratio (1)Not applicable tests (14)Passing tests (2)

IMO, the contrast here is totally fine. Black on light grey is fine. 

That said, I am running into some issues with keyboard navigation (without a screen reader on). I can't seem to navigate and collapse / uncollapse the content with just the keyboard. I also don't have a clear visual focus state. 

I tested this with VoiceOver on Mac and ChromeVox on Chrome OS. The latter seemed to work fine, but with VoiceOver, I wasn't able to access the uncollapsed text content. We should look into why this might be the case, specifically on this screen reader / browser combo.

**Comment by alice** [comment]
No ARIA roles/properties applied. Suggest `button` - demo of `button` pattern here http://thepaciellogroup.github.io/disclosure-button/example2.html or here http://www.heydonworks.com/accessible-web-component-with-aria/ (variation on the same theme).

Element is not focusable at all, and there is no keyboard behaviour defined.

**Comment by notwaldorf** [comment]
Fixed in https://github.com/PolymerElements/iron-collapse/issues/6

**Comment by lpalmaro** [comment]
The keyboard navigation seems to be working fine now. 

With VoiceOver on Mac, I'm not able to get to the uncollapsed content for Collapse #3. I can access the content in 1 and 2 though...

**Comment by punzki** [comment]
I opened https://github.com/PolymerElements/iron-collapse/issues/10. I do not believe that the iron-collapse element itself should be navigable with the keyboard, as toggling it is usually controlled by another element, which is the one that should be navigable.
